server/graphql/v2/interface/Account.js

```diff
@@ -568,6 +569,43 @@ const accountFieldsDefinition = () => ({
       };
     },
   },
+  childrenAccounts: {
+    type: AccountCollection,
+    args: {
+      limit: { type: GraphQLInt, defaultValue: 100 },
+      offset: { type: GraphQLInt, defaultValue: 0 },
+      accountType: {
+        type: new GraphQLList(AccountType),
+      },
+    },
+    async resolve(account, args, req) {
+      if (!req.remoteUser?.isAdminOfCollective(account)) {
+        throw new Unauthorized('You need to be logged in as an admin of the account to see its children accounts');
+      }
```

Comment:
Why did you go for this? Doesn't seem necessary.

Reply:
I'll remove this.